### PR TITLE
add fixture for CitigoE iV 2020 style

### PR DESCRIPTION
**Pull request by bjacquot**
maybe it could be useful

### CODE REVIEW

**Comment by WebSpider** [comment]
Thanks for the fixture!

It looks like the fixture is breaking some tests, interesting!

**Comment by WebSpider** [comment]
Can you please re-run the fixtures with the most recent version of the software? Thanks!

**Comment by bjacquot** [comment]
of course, but i'm not sure about what you mean.
mySkoda software version or homeassistant ?

[myskoda-01JDY9SG0VX8R8ZT1TYYZCKHJR-Škoda CitigoE iV-6915d8dd7c8fac5b35cad8ea765fa096.json](https://github.com/user-attachments/files/18196612/myskoda-01JDY9SG0VX8R8ZT1TYYZCKHJR-Skoda.CitigoE.iV-6915d8dd7c8fac5b35cad8ea765fa096.json)

File: fixtures/citigoe_iv_2020_ne14e2_style.yaml [hunk 1]
```diff
@@ -0,0 +1,772 @@
...
+      "overall_average_gas_consumption": null,
+      "overall_average_mileage_in_km": 62,
+      "overall_average_speed_in_kmph": 41,
+      "overall_average_travel_time_in_min": 94,
+      "overall_mileage_in_km": 186,
+      "overall_travel_time_in_min": 283
+    },
+    "error": null
+  }
+]
```
Comment (prvakt) [inline fixtures/citigoe_iv_2020_ne14e2_style.yaml:772]:
```suggestion
name: CitigoE iV
description: Fixtures for CitigoE iV STYLE 2020
generation_time: '2024-12-19T12:26:16.200782+00:00'
library_version: 0.15.4
vehicles:
  - id: 0
    device_platform: MBB
    system_model_id: NE14E2
    model: CitigoE iV
    model_year: '2020'
    trim_level: STYLE
    software_version: null
    capabilities:
      - id: AUTOMATION
        statuses: []
      - id: DEPARTURE_TIMERS
        statuses: []
      - id: MEASUREMENTS
        statuses: []
      - id: PARKING_POSITION
        statuses: []
      - id: STATE
        statuses: []
      - id: TRIP_STATISTICS
        statuses: []
      - id: VEHICLE_HEALTH_INSPECTION
        statuses: []
      - id: WARNING_LIGHTS
        statuses: []
      - id: VEHICLE_WAKE_UP
        statuses: []
      - id: LOYALTY_PROGRAM
        statuses: []
      - id: POWERPASS_TARIFFS
        statuses: []
      - id: AIR_CONDITIONING_HEATING_SOURCE_ELECTRIC
        statuses: []
      - id: SUBSCRIPTIONS
        statuses: []
      - id: CHARGING_MQB
        statuses: []
      - id: LOYALTY_PROGRAM_WORLDWIDE
        statuses: []
      - id: OUTSIDE_TEMPERATURE
        statuses: []
      - id: PAY_TO_PARK
        statuses: []
      - id: SERVICE_PARTNER
        statuses: []
      - id: CHARGING
        statuses: []
      - id: MISUSE_PROTECTION
        statuses: []
      - id: AIR_CONDITIONING
        statuses: []
reports:
  - type: get
    vehicle_id: 0
    success: true
    endpoint: info
    raw: >-
      {"vin": "TMOCKAA0AA000000", "name": "Example Car", "workshopModeEnabled":
      false, "state": "ACTIVATED", "devicePlatform": "MBB", "specification":
      {"title": "\u0160koda CitigoE iV", "manufacturingDate": "2019-12-10",
      "model": "CitigoE iV", "modelYear": "2020", "body": "Hatchback",
      "trimLevel": "STYLE", "systemCode": "UNKNOWN", "systemModelId": "NE14E2",
      "battery": {"capacityInKWh": 36}, "engine": {"powerInKW": 61}, "gearbox":
      {"type": "E1F"}}, "servicePartner": {"servicePartnerId": "DEU11111"},
      "renders": [], "compositeRenders": [{"layers": [{"url":
      "https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 0}], "viewType":
      "UNMODIFIED_EXTERIOR_SIDE"}, {"layers": [{"url":
      "https://mspgwlivestorage.blob.core.windows.net/renders/decorations/ecitigo_NE_christmas_tree_v1.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 0}, {"url":
      "https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 1}],
      "modifications": {"adjustSpaceInPx": {"top": -316, "bottom": -204, "left":
      -282, "right": -316}, "densityIndependentHeight": 270, "flipHorizontal":
      false, "anchorTo": "LEFT"}, "viewType": "HOME"}, {"layers": [{"url":
      "https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 0}, {"url":
      "https://mspgwlivestorage.blob.core.windows.net/renders/cables/ecitigo_NE_cable_charging_light_v1.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 1}],
      "modifications": {"adjustSpaceInPx": {"top": -316, "bottom": -204, "left":
      -282, "right": -316}, "densityIndependentHeight": 270, "flipHorizontal":
      true, "anchorTo": "LEFT"}, "viewType": "CHARGING_LIGHT"}, {"layers":
      [{"url":
      "https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 0}, {"url":
      "https://mspgwlivestorage.blob.core.windows.net/renders/cables/ecitigo_NE_cable_charging_dark_v1.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 1}],
      "modifications": {"adjustSpaceInPx": {"top": -316, "bottom": -204, "left":
      -282, "right": -316}, "densityIndependentHeight": 270, "flipHorizontal":
      true, "anchorTo": "LEFT"}, "viewType": "CHARGING_DARK"}, {"layers":
      [{"url":
      "https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 0}, {"url":
      "https://mspgwlivestorage.blob.core.windows.net/renders/cables/ecitigo_NE_cable_plugged_in_light_v1.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 1}],
      "modifications": {"adjustSpaceInPx": {"top": -316, "bottom": -204, "left":
      -282, "right": -316}, "densityIndependentHeight": 270, "flipHorizontal":
      true, "anchorTo": "LEFT"}, "viewType": "PLUGGED_IN_LIGHT"}, {"layers":
      [{"url":
      "https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 0}, {"url":
      "https://mspgwlivestorage.blob.core.windows.net/renders/cables/ecitigo_NE_cable_plugged_in_dark_v1.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 1}],
      "modifications": {"adjustSpaceInPx": {"top": -316, "bottom": -204, "left":
      -282, "right": -316}, "densityIndependentHeight": 270, "flipHorizontal":
      true, "anchorTo": "LEFT"}, "viewType": "PLUGGED_IN_DARK"}],
      "capabilities": {"capabilities": [{"id": "AUTOMATION", "statuses": []},
      {"id": "DEPARTURE_TIMERS", "statuses": []}, {"id": "MEASUREMENTS",
      "statuses": []}, {"id": "PARKING_POSITION", "statuses": []}, {"id":
      "STATE", "statuses": []}, {"id": "TRIP_STATISTICS", "statuses": []},
      {"id": "VEHICLE_HEALTH_INSPECTION", "statuses": []}, {"id":
      "WARNING_LIGHTS", "statuses": []}, {"id": "VEHICLE_WAKE_UP", "statuses":
      []}, {"id": "LOYALTY_PROGRAM", "statuses": []}, {"id":
      "POWERPASS_TARIFFS", "statuses": []}, {"id":
      "AIR_CONDITIONING_HEATING_SOURCE_ELECTRIC", "statuses": []}, {"id":
      "SUBSCRIPTIONS", "statuses": []}, {"id": "CHARGING_MQB", "statuses": []},
      {"id": "LOYALTY_PROGRAM_WORLDWIDE", "statuses": []}, {"id":
      "OUTSIDE_TEMPERATURE", "statuses": []}, {"id": "PAY_TO_PARK", "statuses":
      []}, {"id": "SERVICE_PARTNER", "statuses": []}, {"id": "CHARGING",
      "statuses": []}, {"id": "MISUSE_PROTECTION", "statuses": []}, {"id":
      "AIR_CONDITIONING", "statuses": []}]}}
    url: >-
      /v2/garage/vehicles/TMOCKAA0AA000000?connectivityGenerations=MOD1&connectivityGenerations=MOD2&connectivityGenerations=MOD3&connectivityGenerations=MOD4
    result:
      state: ACTIVATED
      specification:
        body: Hatchback
        engine:
          power: 61
          type: null
          capacity_in_liters: null
        model: CitigoE iV
        title: Škoda CitigoE iV
        manufacturing_date: '2019-12-10'
        model_year: '2020'
        system_code: UNKNOWN
        system_model_id: NE14E2
        battery:
          capacity: 36
        max_charging_power: null
        trim_level: STYLE
      vin: TMOCKAA0AA000000
      name: Example Car
      capabilities:
        capabilities:
          - id: AUTOMATION
            statuses: []
          - id: DEPARTURE_TIMERS
            statuses: []
          - id: MEASUREMENTS
            statuses: []
          - id: PARKING_POSITION
            statuses: []
          - id: STATE
            statuses: []
          - id: TRIP_STATISTICS
            statuses: []
          - id: VEHICLE_HEALTH_INSPECTION
            statuses: []
          - id: WARNING_LIGHTS
            statuses: []
          - id: VEHICLE_WAKE_UP
            statuses: []
          - id: LOYALTY_PROGRAM
            statuses: []
          - id: POWERPASS_TARIFFS
            statuses: []
          - id: AIR_CONDITIONING_HEATING_SOURCE_ELECTRIC
            statuses: []
          - id: SUBSCRIPTIONS
            statuses: []
          - id: CHARGING_MQB
            statuses: []
          - id: LOYALTY_PROGRAM_WORLDWIDE
            statuses: []
          - id: OUTSIDE_TEMPERATURE
            statuses: []
          - id: PAY_TO_PARK
            statuses: []
          - id: SERVICE_PARTNER
            statuses: []
          - id: CHARGING
            statuses: []
          - id: MISUSE_PROTECTION
            statuses: []
          - id: AIR_CONDITIONING
            statuses: []
      renders: []
      device_platform: MBB
      workshop_mode_enabled: false
      composite_renders:
        - layers:
            - url: >-
                https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png
              type: REAL
              order: 0
              view_point: exterior_side
          view_type: UNMODIFIED_EXTERIOR_SIDE
        - layers:
            - url: >-
                https://mspgwlivestorage.blob.core.windows.net/renders/decorations/ecitigo_NE_christmas_tree_v1.png
              type: REAL
              order: 0
              view_point: exterior_side
            - url: >-
                https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png
              type: REAL
              order: 1
              view_point: exterior_side
          view_type: HOME
        - layers:
            - url: >-
                https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png
              type: REAL
              order: 0
              view_point: exterior_side
            - url: >-
                https://mspgwlivestorage.blob.core.windows.net/renders/cables/ecitigo_NE_cable_charging_light_v1.png
              type: REAL
              order: 1
              view_point: exterior_side
          view_type: CHARGING_LIGHT
        - layers:
            - url: >-
                https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png
              type: REAL
              order: 0
              view_point: exterior_side
            - url: >-
                https://mspgwlivestorage.blob.core.windows.net/renders/cables/ecitigo_NE_cable_charging_dark_v1.png
              type: REAL
              order: 1
              view_point: exterior_side
          view_type: CHARGING_DARK
        - layers:
            - url: >-
                https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png
              type: REAL
              order: 0
              view_point: exterior_side
            - url: >-
                https://mspgwlivestorage.blob.core.windows.net/renders/cables/ecitigo_NE_cable_plugged_in_light_v1.png
              type: REAL
              order: 1
              view_point: exterior_side
          view_type: PLUGGED_IN_LIGHT
        - layers:
            - url: >-
                https://iprenders.blob.core.windows.net/basene1s20201108/B4A1KSGw9WT3BZCM6j0.ysmOnq-JHCLOQGSE-19201080dayvext_side1080.png
              type: REAL
              order: 0
              view_point: exterior_side
            - url: >-
                https://mspgwlivestorage.blob.core.windows.net/renders/cables/ecitigo_NE_cable_plugged_in_dark_v1.png
              type: REAL
              order: 1
              view_point: exterior_side
          view_type: PLUGGED_IN_DARK
      service_partner:
        id: DEU11111
      software_version: null
      license_plate: null
      errors: null
    error: null
  - type: get
    vehicle_id: 0
    success: true
    endpoint: status
    raw: >-
      {"overall": {"doorsLocked": "NO", "locked": "NO", "doors": "CLOSED",
      "windows": "UNSUPPORTED", "lights": "OFF"}, "detail": {"sunroof":
      "UNSUPPORTED", "trunk": "CLOSED", "bonnet": "CLOSED"}, "renders":
      {"lightMode": {"oneX":
      "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=1x&theme=LIGHT",
      "oneAndHalfX":
      "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=1-5x&theme=LIGHT",
      "twoX":
      "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=2x&theme=LIGHT",
      "threeX":
      "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=3x&theme=LIGHT"},
      "darkMode": {"oneX":
      "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=1x&theme=DARK",
      "oneAndHalfX":
      "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=1-5x&theme=DARK",
      "twoX":
      "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=2x&theme=DARK",
      "threeX":
      "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=3x&theme=DARK"}},
      "carCapturedTimestamp": "2024-12-19T10:51:07Z"}
    url: /v2/vehicle-status/TMOCKAA0AA000000
    result:
      detail:
        bonnet: CLOSED
        sunroof: UNSUPPORTED
        trunk: CLOSED
      overall:
        doors: CLOSED
        doors_locked: 'NO'
        lights: 'OFF'
        locked: 'NO'
        windows: UNSUPPORTED
      renders:
        light_mode:
          one_x: >-
            https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=1x&theme=LIGHT
          one_and_half_x: >-
            https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=1-5x&theme=LIGHT
          two_x: >-
            https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=2x&theme=LIGHT
          three_x: >-
            https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=3x&theme=LIGHT
        dark_mode:
          one_x: >-
            https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=1x&theme=DARK
          one_and_half_x: >-
            https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=1-5x&theme=DARK
          two_x: >-
            https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=2x&theme=DARK
          three_x: >-
            https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=HATCHBACK&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1733938917&dimension=3x&theme=DARK
      car_captured_timestamp: '2024-12-19T10:51:07+00:00'
    error: null
  - type: get
    vehicle_id: 0
    success: true
    endpoint: air_conditioning
    raw: >-
      {"state": "OFF", "runningRequests": [], "targetTemperature":
      {"temperatureValue": 20.0, "unitInCar": "CELSIUS"},
      "airConditioningWithoutExternalPower": false, "steeringWheelPosition":
      "LEFT", "chargerConnectionState": "DISCONNECTED", "chargerLockState":
      "LOCKED", "windowHeatingState": {"front": "OFF", "rear": "OFF",
      "unspecified": "INVALID"}, "timers": [], "carCapturedTimestamp":
      "2024-12-18T18:19:19Z", "outsideTemperature": {"temperatureValue": 13.5,
      "temperatureUnit": "CELSIUS", "carCapturedTimestamp":
      "2024-12-19T12:26:06Z"}, "errors": [{"type":
      "UNAVAILABLE_VEHICLE_INFORMATION", "description": "APIs for obtaining seat
      heating status are not available"}, {"type":
      "UNAVAILABLE_TIMERS_INFORMATION", "description": "APIs for obtaining air
      conditioning timers are not available"}]}
    url: /v2/air-conditioning/TMOCKAA0AA000000
    result:
      timers: []
      errors:
        - type: UNAVAILABLE_VEHICLE_INFORMATION
          description: APIs for obtaining seat heating status are not available
        - type: UNAVAILABLE_TIMERS_INFORMATION
          description: APIs for obtaining air conditioning timers are not available
      state: 'OFF'
      steering_wheel_position: LEFT
      window_heating_state:
        front: 'OFF'
        rear: 'OFF'
        unspecified: INVALID
      car_captured_timestamp: '2024-12-18T18:19:19+00:00'
      air_conditioning_at_unlock: null
      charger_connection_state: DISCONNECTED
      charger_lock_state: LOCKED
      estimated_date_time_to_reach_target_temperature: null
      heater_source: null
      seat_heating_activated: null
      target_temperature:
        temperature_value: 20
        unit_in_car: CELSIUS
      window_heating_enabled: null
      air_conditioning_without_external_power: false
      outside_temperature:
        temperature_value: 13.5
        unit_in_car: CELSIUS
        car_captured_timestamp: '2024-12-19T12:26:06+00:00'
    error: null
  - type: get
    vehicle_id: 0
    success: false
    endpoint: auxiliary_heating
    raw: null
    url: null
    result: null
    error: >
      Traceback (most recent call last):
        File "/usr/local/lib/python3.13/site-packages/myskoda/myskoda.py", line 395, in generate_fixture_report
          result = await self.get_endpoint(vin, endpoint, anonymize=True)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/local/lib/python3.13/site-packages/myskoda/myskoda.py", line 440, in get_endpoint
          return await method(vin, anonymize=anonymize)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/local/lib/python3.13/site-packages/myskoda/rest_api.py", line 181, in get_auxiliary_heating
          data=await self._make_get_request(url),
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/local/lib/python3.13/site-packages/myskoda/rest_api.py", line 104, in _make_get_request
          return await self._make_request(url=url, method="GET")
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/local/lib/python3.13/site-packages/myskoda/rest_api.py", line 94, in _make_request
          response.raise_for_status()
          ~~~~~~~~~~~~~~~~~~~~~~~~~^^
        File "/usr/local/lib/python3.13/site-packages/aiohttp/client_reqrep.py", line 1157, in raise_for_status
          raise ClientResponseError(
          ...<5 lines>...
          )
      aiohttp.client_exceptions.ClientResponseError: 500, message='Internal
      Server Error',
      url='https://mysmob.api.connect.skoda-auto.cz/api/v2/air-conditioning/TMOCKAA0AA000000/auxiliary-heating'
  - type: get
    vehicle_id: 0
    success: true
    endpoint: positions
    raw: >-
      {"positions": [{"type": "VEHICLE", "gpsCoordinates": {"latitude":
      53.470636, "longitude": 9.689872}, "address": {"city": "Example City",
      "street": "Example Avenue", "houseNumber": "15", "zipCode": "54321",
      "countryCode": "DEU"}}], "errors": []}
    url: /v1/maps/positions?vin=TMOCKAA0AA000000
    result:
      errors: []
      positions:
        - address:
            country_code: DEU
            zip_code: '54321'
            house_number: '15'
            street: Example Avenue
            city: Example City
            country: null
          gps_coordinates:
            latitude: 53.470636
            longitude: 9.689872
          type: VEHICLE
    error: null
  - type: get
    vehicle_id: 0
    success: true
    endpoint: health
    raw: >-
      {"capturedAt": "2024-12-19T12:26:06Z", "mileageInKm": 50137,
      "warningLights": [{"category": "ASSISTANCE", "defects": []}, {"category":
      "COMFORT", "defects": []}, {"category": "BRAKE", "defects": []},
      {"category": "ENGINE", "defects": []}, {"category": "LIGHTING", "defects":
      []}, {"category": "TIRE", "defects": []}, {"category": "OTHER", "defects":
      []}]}
    url: /v1/vehicle-health-report/warning-lights/TMOCKAA0AA000000
    result:
      warning_lights:
        - category: ASSISTANCE
          defects: []
        - category: COMFORT
          defects: []
        - category: BRAKE
          defects: []
        - category: ENGINE
          defects: []
        - category: LIGHTING
          defects: []
        - category: TIRE
          defects: []
        - category: OTHER
          defects: []
      mileage_in_km: 50137
      captured_at: '2024-12-19T12:26:06+00:00'
    error: null
  - type: get
    vehicle_id: 0
    success: true
    endpoint: charging
    raw: >-
      {"isVehicleInSavedLocation": false, "status":
      {"remainingTimeToFullyChargedInMinutes": 80, "state": "CONNECT_CABLE",
      "battery": {"remainingCruisingRangeInMeters": 137000,
      "stateOfChargeInPercent": 75}}, "settings": {"availableChargeModes": [],
      "maxChargeCurrentAc": "MAXIMUM"}, "carCapturedTimestamp":
      "2024-12-19T10:50:24Z", "errors": [{"type": "CARE_MODE_IS_NOT_AVAILABLE",
      "description": "Care mode is not available."}, {"type":
      "AUTO_UNLOCK_IS_NOT_AVAILABLE", "description": "Auto unlock is not
      available."}, {"type": "CHARGE_LIMIT_IS_NOT_AVAILABLE", "description":
      "Charge limit is not available."}]}
    url: /v1/charging/TMOCKAA0AA000000
    result:
      errors:
        - type: CARE_MODE_IS_NOT_AVAILABLE
          description: Care mode is not available.
        - type: AUTO_UNLOCK_IS_NOT_AVAILABLE
          description: Auto unlock is not available.
        - type: CHARGE_LIMIT_IS_NOT_AVAILABLE
          description: Charge limit is not available.
      settings:
        available_charge_modes: []
        max_charge_current_ac: MAXIMUM
        auto_unlock_plug_when_charged: null
        battery_support: null
        charging_care_mode: null
        preferred_charge_mode: null
        target_state_of_charge_in_percent: null
      is_vehicle_in_saved_location: false
      car_captured_timestamp: '2024-12-19T10:50:24+00:00'
      status:
        battery:
          state_of_charge_in_percent: 75
          remaining_cruising_range_in_meters: 137000
        state: CONNECT_CABLE
        charge_power_in_kw: null
        charging_rate_in_kilometers_per_hour: null
        charge_type: null
        errors: null
        remaining_time_to_fully_charged_in_minutes: 80
    error: null
  - type: get
    vehicle_id: 0
    success: true
    endpoint: maintenance
    raw: >-
      {"maintenanceReport": {"capturedAt": "2024-12-19T11:26:05Z",
      "inspectionDueInDays": 404, "inspectionDueInKm": 16500, "mileageInKm":
      50137}, "preferredServicePartner": {"name": "Example Service Partner",
      "brand": "C", "partnerNumber": "1111", "id": "DEU11111", "contact":
      {"phone": "+49 1234 567890", "url": "https://example.com", "email":
      "user@example.com"}, "address": {"city": "Example City", "street":
      "Example Avenue", "houseNumber": "15", "zipCode": "54321", "countryCode":
      "DEU"}, "location": {"latitude": 53.470636, "longitude": 9.689872},
      "openingHours": [{"periodStart": "MONDAY", "periodEnd": "THURSDAY",
      "openingTimes": [{"from": "08:00:00", "to": "12:00:00"}, {"from":
      "14:00:00", "to": "18:00:00"}]}, {"periodStart": "FRIDAY", "periodEnd":
      "FRIDAY", "openingTimes": [{"from": "08:00:00", "to": "12:00:00"},
      {"from": "14:00:00", "to": "17:30:00"}]}, {"periodStart": "SATURDAY",
      "periodEnd": "SATURDAY", "openingTimes": [{"from": "08:15:00", "to":
      "12:00:00"}]}, {"periodStart": "SUNDAY", "periodEnd": "SUNDAY",
      "openingTimes": []}]}}
    url: /v3/vehicle-maintenance/vehicles/TMOCKAA0AA000000
    result:
      maintenance_report:
        captured_at: '2024-12-19T11:26:05+00:00'
        mileage_in_km: 50137
        inspection_due_in_days: 404
        inspection_due_in_km: 16500
        oil_service_due_in_days: null
        oil_service_due_in_km: null
      predictive_maintenance: null
      preferred_service_partner:
        address:
          country_code: DEU
          zip_code: '54321'
          house_number: '15'
          street: Example Avenue
          city: Example City
          country: null
        brand: C
        contact:
          email: user@example.com
          phone: +49 1234 567890
          url: 'https://example.com'
        id: DEU11111
        location:
          latitude: 53.470636
          longitude: 9.689872
        name: Example Service Partner
        opening_hours:
          - opening_times:
              - start: '08:00:00'
                end: '12:00:00'
              - start: '14:00:00'
                end: '18:00:00'
            period_end: THURSDAY
            period_start: MONDAY
          - opening_times:
              - start: '08:00:00'
                end: '12:00:00'
              - start: '14:00:00'
                end: '17:30:00'
            period_end: FRIDAY
            period_start: FRIDAY
          - opening_times:
              - start: '08:15:00'
                end: '12:00:00'
            period_end: SATURDAY
            period_start: SATURDAY
          - opening_times: []
            period_end: SUNDAY
            period_start: SUNDAY
        partner_number: '1111'
    error: null
  - type: get
    vehicle_id: 0
    success: true
    endpoint: driving_range
    raw: >-
      {"carType": "electric", "totalRangeInKm": 137, "primaryEngineRange":
      {"engineType": "electric", "currentSoCInPercent": 75,
      "remainingRangeInKm": 137}, "carCapturedTimestamp":
      "2024-12-19T12:26:06Z"}
    url: /v2/vehicle-status/TMOCKAA0AA000000/driving-range
    result:
      car_captured_timestamp: '2024-12-19T12:26:06+00:00'
      car_type: electric
      primary_engine_range:
        engine_type: electric
        current_fuel_level_in_percent: null
        current_soc_in_percent: 75
        remaining_range_in_km: 137
      secondary_engine_range: null
      total_range_in_km: 137
      ad_blue_range: null
    error: null
  - type: get
    vehicle_id: 0
    success: true
    endpoint: trip_statistics
    raw: >-
      {"overallMileageInKm": 186, "overallTravelTimeInMin": 283,
      "overallAverageMileageInKm": 47, "overallAverageTravelTimeInMin": 71,
      "overallAverageElectricConsumption": 14.26, "overallAverageSpeedInKmph":
      41, "vehicleType": "ELECTRIC", "detailedStatistics": [{"date":
      "2024-12-16", "tripIds": ["2831108042", "2831012870", "2830769588"],
      "mileageInKm": 59, "travelTimeInMin": 89, "averageElectricConsumption":
      13.95, "averageRecuperation": 3.49, "averageAuxConsumption": 2.93,
      "averageSpeedInKmph": 42}, {"date": "2024-12-17", "tripIds":
      ["2831438620", "2831222124"], "mileageInKm": 84, "travelTimeInMin": 120,
      "averageElectricConsumption": 14.77, "averageRecuperation": 3.11,
      "averageAuxConsumption": 3.86, "averageSpeedInKmph": 43}, {"date":
      "2024-12-18", "tripIds": ["2831945865", "2831833733"], "mileageInKm": 43,
      "travelTimeInMin": 74, "averageElectricConsumption": 13.69,
      "averageRecuperation": 5.28, "averageAuxConsumption": 4.35,
      "averageSpeedInKmph": 36}, {"date": "2024-12-19"}, {"date": "2024-12-20"},
      {"date": "2024-12-21"}, {"date": "2024-12-22"}]}
    url: >-
      /v1/trip-statistics/TMOCKAA0AA000000?offsetType=week&offset=0&timezone=Europe%2FBerlin
    result:
      vehicle_type: ELECTRIC
      detailed_statistics:
        - date: '2024-12-16'
          average_fuel_consumption: null
          average_gas_consumption: null
          average_speed_in_kmph: 42
          average_electric_consumption: 13.95
          average_recuperation: 3.49
          average_aux_consumption: 2.93
          mileage_in_km: 59
          travel_time_in_min: 89
          trip_ids:
            - 2831108042
            - 2831012870
            - 2830769588
        - date: '2024-12-17'
          average_fuel_consumption: null
          average_gas_consumption: null
          average_speed_in_kmph: 43
          average_electric_consumption: 14.77
          average_recuperation: 3.11
          average_aux_consumption: 3.86
          mileage_in_km: 84
          travel_time_in_min: 120
          trip_ids:
            - 2831438620
            - 2831222124
        - date: '2024-12-18'
          average_fuel_consumption: null
          average_gas_consumption: null
          average_speed_in_kmph: 36
          average_electric_consumption: 13.69
          average_recuperation: 5.28
          average_aux_consumption: 4.35
          mileage_in_km: 43
          travel_time_in_min: 74
          trip_ids:
            - 2831945865
            - 2831833733
        - date: '2024-12-19'
          average_fuel_consumption: null
          average_gas_consumption: null
          average_speed_in_kmph: null
          average_electric_consumption: null
          average_recuperation: null
          average_aux_consumption: null
          mileage_in_km: null
          travel_time_in_min: null
          trip_ids: null
        - date: '2024-12-20'
          average_fuel_consumption: null
          average_gas_consumption: null
          average_speed_in_kmph: null
          average_electric_consumption: null
          average_recuperation: null
          average_aux_consumption: null
          mileage_in_km: null
          travel_time_in_min: null
          trip_ids: null
        - date: '2024-12-21'
          average_fuel_consumption: null
          average_gas_consumption: null
          average_speed_in_kmph: null
          average_electric_consumption: null
          average_recuperation: null
          average_aux_consumption: null
          mileage_in_km: null
          travel_time_in_min: null
          trip_ids: null
        - date: '2024-12-22'
          average_fuel_consumption: null
          average_gas_consumption: null
          average_speed_in_kmph: null
          average_electric_consumption: null
          average_recuperation: null
          average_aux_consumption: null
          mileage_in_km: null
          travel_time_in_min: null
          trip_ids: null
      overall_average_electric_consumption: 14.26
      overall_average_fuel_consumption: null
      overall_average_gas_consumption: null
      overall_average_mileage_in_km: 47
      overall_average_speed_in_kmph: 41
      overall_average_travel_time_in_min: 71
      overall_mileage_in_km: 186
      overall_travel_time_in_min: 283
    error: null
```

**Review by prvakt — CHANGES_REQUESTED** [review]
json must be converted to yaml.. please use my suggested changes, it should work afterwards

**Review by prvakt — APPROVED** [review]

**Comment by prvakt** [comment]
I was able to fix it within this PR myself, but thanks for the fixtures !

**Comment by bjacquot** [comment]
thanks

**Pull request merged**